### DISCUSSION TITLE
How to connect to a self-hosted Qfieldcloud from Qfield

### DISCUSSION THREAD

**Discussion by JUJUBAEDATE** [opengisch/QField#5034] [Q&A]
Following the readme, I installed Qfieldcloud on my PC (Win 11, Intel N100) to use as a server. It seems that CSS is not applied, but I can access the Qfieldcloud Admin page. What is the next step? I want to connect to my cloud from the Qfield app, but I'm not sure how to do it. Do I need to modify the source code of the Qfield app and rebuild the app for myself?
![qfieldcloud self host](https://github.com/opengisch/QField/assets/105536224/61558020-00a7-41a0-bfcd-b8155984d9ce)

**Comment by meyerlor** [comment]
have you found out how to connect Qfield to your own QField CLoud instance?

**Comment by 9ls1** [comment]
@JUJUBAEDATE Not sure if this is what you're looking for? https://github.com/opengisch/QField/discussions/2927#discussioncomment-2971579: "On QGIS and QField, double tap the QFieldCloud logo to change the API URL to your own instance."
Change the Server URL.
![image](https://github.com/opengisch/QField/assets/10632163/1bdfa5d7-afff-424c-a0b9-fd1819273bdd)
